Packs/Gatewatcher-AionIQ/Integrations/GCenter/README.md

```diff
@@ -21,6 +21,11 @@ To simplify GCenter v2.5.3.102 is called GCenter in the Pack.
 ## Commands
 You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
 After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### gw-list-alerts (deprecated)
+***
+List all alerts. The command is deprecated. You should use the the `gw-es-wrapper` command instead.
```

Comment:
Why is a deprecated command needed in the readme? If neede, it should be added with an example, otherwise it is less helpfull.

Reply:
Hello,
the deprecated command is documented here just to reference the new one (`gw-es-wrapper`) so let me know if you want to remove it.

Reply:
I think it is best to remove it, but change the command's description in the yml file to `List all alerts (Deprecated. use gw-es-wrapper command instead.`